src/components/Header/SideNav/Dropdown.js

```diff
@@ -5,20 +5,16 @@ import is from 'styled-is'
 import Chevron from '../../Common/Chevron'
 import InnerAnchorItem from './InnerAnchorItem'
 import headerItemStyles from '../headerItemStyles'
-import sideNavItemSpacing from './sideNavItemSpacing'
-import {
-  fontSizeAndWeight,
-  defaultStyles,
-  hoverActiveStyles
-} from './outerItemStyles'
+import { fontSizeAndWeight, itemSpacing } from './sideNavItemStyles'
```

Comment:
not to nag, but since we only export two objects on the `sideNavItemStyles`, and we use them together everytime, it would simplify to just export `default` a single object with all the rules and then use it everywhere?

Reply:
If this logic could be applied on any other file, even better..!

Reply:
Oh yeah good point. Something like `const listItemStyles` that encapsulates both :+1: 

Well, if you don't like the idea of having files that are purely for common styling, we could pick a component to "own" these styles. Say, we could:
1. put this `listItemStyles` inside `OuterAnchorItem.js` & export from there
2. delete the extra `listItemStyles.js` file
3. `import { listItemStyles } from 'OuterAnchorItem'` within InnerAnchorItem & Dropdown.js

If it's just a matter of not having more files than the number of actual components, then we could do that?